### PR TITLE
update docs to remove Openshift references

### PR DESCRIPTION
**Pull request by tobybellwood**
Now that we have Kubernetes Aliases in the API, we should use them

(and other tidyups)

### CODE REVIEW

File: docs/administering-lagoon/graphql-queries.md [hunk 1]
```diff
@@ -211,25 +211,25 @@ Now for every deployment you will receive messages in your defined channel.
 
 ## Example GraphQL queries
 
-### Adding a new OpenShift target
+### Adding a new Kubernetes target
 
 !!! Note "Note:"
-    In Lagoon 1.x `addOpenshift` is used for both OpenShift and Kubernetes targets. In Lagoon 2.x this will change.
+    In Lagoon, both `addOpenshift` and `addKubernetes` can be used for both OpenShift and Kubernetes targets - either will work interchangeably.
```
Comment (shreddedbacon) [inline docs/administering-lagoon/graphql-queries.md]: Swap the order of these, so openshift is second?

File: docs/docker-images/mongodb.md [hunk 2]
```diff
@@ -8,8 +8,8 @@
...
 This Dockerfile is intended to be used to set up a standalone MongoDB database server.
 
-## Lagoon & OpenShift adaptions
+## Lagoon adaptions
 
-This image is prepared to be used on Lagoon, which leverages OpenShift. There are therefore some things already done:
+This image is prepared to be used on Lagoon. There are therefore some things already done:
 
-* Folder permissions are automatically adapted with [`fix-permissions`](https://github.com/uselagoon/lagoon-images/blob/main/images/commons/fix-permissions), so this image will work with a random user, and therefore also on OpenShift.
+* Folder permissions are automatically adapted with [`fix-permissions`](https://github.com/uselagoon/lagoon-images/blob/main/images/commons/fix-permissions), so this image will work with a random user, and therefore also on OpenShift/Kubernetes.
```
Comment (shreddedbacon) [inline docs/docker-images/mongodb.md]: Another swap the ordering?

File: docs/using-lagoon-the-basics/lagoon-yml.md [hunk 3]
```diff
@@ -457,7 +457,7 @@ With the key `ssh` you can define another SSH endpoint that should be used by th
 
 ### `additional-yaml`
 
-The `additional-yaml` has some super powers. Basically, it allows you to define any arbitrary YAML configuration file to be inserted before the build step \(it still needs to be valid Kubernetes/OpenShift YAML, though ☺\).
+The `additional-yaml` has some super powers. Basically, it allows you to define any arbitrary YAML configuration file to be inserted before the build step \(it still needs to be valid Kubernetes YAML, though ☺\).
```
Comment (shreddedbacon) [inline docs/using-lagoon-the-basics/lagoon-yml.md]: Drop additional-yaml, it isn't supported anymore, see #3136

**Review by shreddedbacon — CHANGES_REQUESTED** [review]
Few things to check

**Comment by rocketeerbkw** [comment]
> and other tidyups

image

[nmane](https://github.com/uselagoon/lagoon/blob/main/docs/using-lagoon-the-basics/setup-project.md?plain=1#L11) should be name

**Review by shreddedbacon — APPROVED** [review]

**Pull request merged**